lib/Mojo/BaseUtil.pm

```diff
@@ -0,0 +1,45 @@
+package Mojo::BaseUtil;
+
+# Only using pure Perl as the only purpose of this module is to break a circular dependency involving Mojo::Base
+use strict;
```

Comment:
Minor detail, I think we should `use v5.16;` here to include the feature bundle as Mojo::Base -strict would. The rest of the things it imports are unlikely to be important here.

Reply:
or alternatively, add `use feature ':5.16';` to match Mojo::Base's declarations.

Reply:
So I now did the same as in lib/Mojo/Base.pm

```
use strict;
use warnings;
use feature ':5.16';
```

good like this?